scripts/record_and_play_trajectories.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python3
+"""Record data while the user is manually moving the robot."""
+import argparse
+import curses
+import numpy as np
+
+# %%
+
+
+# %%
+
+import robot_fingers
+robot = robot_fingers.Robot.create_by_name('trifingerpro')
+robot.initialize()
```

Comment:
Please move these two lines down to the `if __name__`-section.  Otherwise it is executed also if the file is only imported (which is, e.g. done by some analysis tools or documentation builders).

Reply:
done